### PR TITLE
New version

### PR DESCRIPTION
**Pull request by kain88-de**
- remove namespace package
- update to 1.0.2
- rerender

### CODE REVIEW

**Comment by kain88-de** [comment]
@conda-forge-admin, please update circle

**Comment by kain88-de** [comment]
@dotsdl there seems to be problems with python <3.6 do you know where this could be coming from?

**Comment by dotsdl** [comment]
Not really sure what's happening here. Just did an install of `datreant` 1.0.2 with pip on Python 2.7 and it seems to function as expected. Looks like the module isn't found in the Travis runs?

**Comment by kain88-de** [comment]
yeah I get the same. I'm also a bit surprised. Especially since the 3.6
version works.

**Comment by kain88-de** [comment]
@dotsdl do you have more experience with circle ci to try and debug this?

**Comment by kain88-de** [comment]
OK, I'm able to reproduce this on a local VM with Fedora 27. Interestingly with the current version of conda-build only the package itself gets installed for the correct python version. All dependencies are installed in  python3.6 specific directories.

@ocefpaf do you know anything about this. It's a new problem for me. There isn't anything in the recipe that the would suggest doing this. Here are the lines from the [CircleCI build](https://circleci.com/gh/datreant/datreant-feedstock/16?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link) for the last test.

```
The following NEW packages will be INSTALLED:

    asciitree:          0.3.3-py_2         conda-forge
    ca-certificates:    2018.4.16-0        conda-forge
    datreant:           1.0.2-0            local      
    fuzzywuzzy:         0.16.0-py_0        conda-forge
    libgcc-ng:          7.2.0-hdf63c60_3   defaults   
    ncurses:            5.9-10             conda-forge
    openssl:            1.0.2o-0           conda-forge
    pathlib2:           2.3.2-py36_0       conda-forge
    pyparsing:          2.2.0-py_1         conda-forge
    python:             3.6.5-1            conda-forge
    python-levenshtein: 0.12.0-py36_0      conda-forge
    readline:           7.0-0              conda-forge
    scandir:            1.7-py36h470a237_1 conda-forge
    six:                1.11.0-py36_1      conda-forge
    sqlite:             3.20.1-2           conda-forge
    tk:                 8.6.8-0            conda-forge
    xz:                 5.2.3-0            conda-forge
    zlib:               1.2.11-h470a237_3  conda-forge

Preparing transaction: ...working... done
Verifying transaction: ...working... done
Executing transaction: ...working... done
import: 'datreant'
Traceback (most recent call last):
  File "/home/conda/feedstock_root/build_artifacts/datreant_1532271937623/test_tmp/run_test.py", line 2, in <module>
    import datreant
ModuleNotFoundError: No module named 'datreant'
Tests failed for datreant-1.0.2-0.tar.bz2 - moving package to /home/conda/feedstock_root/build_artifacts/broken
WARNING:conda_build.build:Tests failed for datreant-1.0.2-0.tar.bz2 - moving package to /home/conda/feedstock_root/build_artifacts/broken
removing: datreant-1.0.2-0.tar.bz2
TESTS FAILED: datreant-1.0.2-0.tar.bz2
Exited with code 1
```

Notice how it installed python 3.6 in the python 2.7 build?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,29 +1,43 @@
...
+  script: python -m pip install --no-deps --ignore-installed .
   skip: True  # [win]
 
 requirements:
-  build:
+  host:
     - python
+    - pip
   run:
-    - python
```
Comment (ocefpaf) [inline recipe/meta.yaml]: There is no python dependency on run?

Comment (kain88-de) [reply recipe/meta.yaml]: I've just seen that. I don't remember why I removed it.

**Comment by ocefpaf** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml (hunk 1 above)
Comment (kain88-de) [reply recipe/meta.yaml]: Thanks for this. That was the missing link.

File: .travis.yml [hunk 2]
```diff
@@ -8,10 +8,10 @@ osx_image: xcode6.4
 
 env:
   matrix:
-    
-    - CONDA_PY=27
-    - CONDA_PY=35
-    - CONDA_PY=36
+    - CONFIG=osx_python2.7
+    - CONFIG=osx_python3.5
+    - CONFIG=osx_python3.6
```
Comment (ocefpaf) [inline .travis.yml:13]: Now we have an updated Python build matrix. Hopefully this will fix your problem.

Comment (kain88-de) [reply .travis.yml:13]: yes it does. Thank you!

**Pull request merged**